source/routing/next_stop_time.h

```diff
@@ -197,12 +199,28 @@ struct NextStopTime {
     const type::Data& data;
 };
 
+struct CachedNextStopTimeKey {
+    uint32_t from; //first day concerned by the cache
```

Comment:
Initially it is, but it's converted to a Date in  next_stop_time.cpp:457.
Sounded the easiest way to compare without misleading.
And thinking of an hypothetical smaller cache time-window is a too big dev at least for now.